packages/ckeditor5-autoformat/docs/_snippets/features/autoformat.js

```diff
@@ -39,6 +41,7 @@ ClassicEditor
 				'blockQuote',
 				'codeBlock',
 				'horizontalLine',
+				'todolist',
```

Comment:
I'd move it to other lists, just like it is in the to-do list guide. Feels more natural.

Reply:
Done.